sklearn/multiclass.py

```diff
@@ -513,6 +514,11 @@ def partial_fit(self, X, y, classes=None):
                                 range(self.n_classes_ *
                                 (self.n_classes_-1) // 2)]
 
+        if not set(self.classes_).issuperset(y):
+            raise ValueError("Mini-batch contains {0} while classes "
```

Comment:
I would just write `while it must be a subset of`, I don't think the word classes is appropriate here.